flake.nix

```diff
@@ -1,88 +1,77 @@
 {
   description = "Hyprspace";
 
-  # Nixpkgs / NixOS version to use.
-  inputs.nixpkgs.follows = "hyprland/nixpkgs";
+  inputs.hyprland.url = "github:hyprwm/Hyprland";
```

Comment:
If this no longer needs a version pin, I'd update in the README too

Reply:
Sure I'll do that np